src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,60 @@
+        * {
+            margin: 0px;
+            padding: 0px;
+            box-sizing: border-box;
+        }
+        
+        .navbar {
+            position: sticky;
+            left: 0%;
+            right: 0%;
+            top: 0%;
+            height: 80px;
+            background: #FFFFFF;
+            box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+            padding: 0 50px;
+            color: black;
+        }
+        
+        .logo {
+            width: 48px;
+            height: auto;
+            margin: 8px;
+        }
+        
+        .nav-right {
+            width: fit-content;
+            height: 80px;
+        }
+        
+        .nav-tabs {
+            width: fit-content;
+            height: 80px;
+            border: none;
+            margin-right: 0px!important;
+        }
+        
+        .nav-link {
+            width: 104px;
+            font-size: 16px;
+            text-align: center;
+            line-height: 80px;
+            border: none;
+            color: black;
+            padding: 0px;
+        }
+        
+        .active {
+            box-shadow: 0px 2px 0px #873FE2;
```

Comment:
I dont think we need a box shadow for the active links. Check with Cynthia or blendi on this but i dont see that in figma